caffe2/opt/custom/glow_net_transform.cc

```diff
@@ -34,13 +34,13 @@ C10_DEFINE_string(
     "Shape hints in the form of Name:d0,d1:d2;");
 
 C10_DEFINE_string(
-    onnxifi_blacklist,
+    onnxifi_denylist,
```

Comment:
this is not backward-compatible unfortunately. Glow has some code (net_runner) that overrides this gflag. Maybe keep the interface bits intact for now? Or update the Glow code in the same diff

Reply:
Sigh, I hoped to avoid this by excluding `third_party` from my find and replace but I guess it didn't work. Yeah, let's keep the interface bits the same for now.

Reply:
Reverted.